internal/controller/cluster_monitoring_controller.go

```diff
@@ -83,25 +81,29 @@ func (r *ClusterMonitoringReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
+	ctx = context.WithValue(ctx, "cluster", cluster.Name)
```

Comment:
Can you test this to make sure this logs right?

Reply:
Sure

Reply:
It's missing the installation though

Reply:
True, there is no sign of it in any of the operator's logs

Reply:
Should I revert back to the previous code ?

Reply:
I'm not sure we really need the installation so I'd leave as is?

Reply:
I have an idea :D